Extension/src/LanguageServer/configurations.ts

```diff
@@ -1105,7 +1105,7 @@ export class CppProperties {
         }
     }
 
-    private compileCommandsFileWatcherTimer?: NodeJS.Timer;
+    private compileCommandsFileWatcherTimer?: NodeJS.Timeout;
```

Comment:
These were actually mis-typed originally - and now that VSCode's electron is at node 18.15.0 - the interface for `Timer` and `Timeout` are slightly different, and intervals are declared as  `Timeout` and not `Timer`